api/internal/localizer/localizer_test.go

```diff
@@ -185,13 +185,17 @@ func TestLoadGVKNN(t *testing.T) {
 }
 
 func TestLoadLegacyFields(t *testing.T) {
-	// TODO(annasong): add referenced files when implement legacy field localization
 	kustomization := map[string]string{
+		// We test that `kustomize localize` ignores helm charts. The code neither
+		// accesses the referenced file, which does not exist, nor copies it to
```

Comment:
Let's discuss this with @yuwenma and @KnVerey after the holidays. I agree that we should not handle localization of helm fields in alpha, but silently ignoring it doesn't seem to be the right thing to do. I think our options here are to either copy it over as-is, throw a warning, or throw an error if we see use of the helm field.

Reply:
After discussing offline, @natasha41575 and I concluded that this PR can be merged without addressing `helm` fields, as the PR doesn't make any changes specific to `helm`. We'll discuss the particulars after the holidays; in the meantime, I'll try to copy local files to the `localize` destination.